pkgs/development/web/playwright/webkit.nix

```
@@ -0,0 +1,134 @@
+{ stdenv
+, fetchzip
+, fetchFromGitHub
+, wrapGAppsHook
+, autoPatchelfHook
+, patchelfUnstable
+, at-spi2-atk
+, libgcc
+, cairo
+, enchant
+, libepoxy
+, libevent
+, flite
+, fontconfig
+, freetype
+, mesa
+, libglvnd
+, libgcrypt
+, gtk3
+, gdk-pixbuf
+, glib
+, libgpg-error
+, gst_all_1
+, harfbuzzFull
+, harfbuzz
+, hyphen
+, icu70
+, libjpeg8
+, lcms
+, libmanette
+, openjpeg
+, libopus
+, pango
+, libpng
+, libsecret
+, libsoup_3
+, sqlite
+, systemdLibs
+, libtasn1
+, libwebp
+, rigsofrods-bin
+, woff2
+, xorg
+, libxml2
+, libxslt
+, libvpx
+, zlib
+, suffix
+, revision
+}:
+let
+  suffix' = if suffix == "linux"
+            then "ubuntu-22.04"
+            else suffix;
+
+  libvpx_1_12 = libvpx.overrideAttrs (finalAttrs: previousAttrs: {
+    version = "1.12.0";
+    src = fetchFromGitHub {
+      owner = "webmproject";
+      repo = previousAttrs.pname;
```

```suggestion
      repo = "libvpx";
```
If the package is using finalAttrs, then the usage is wrong